### PR TITLE
Add Views 

### PR DESCRIPTION
**Pull request by eaesmat**
**This PR contains the following changes**

- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+      <div class="container border rounded-3 p-3 col-8 justify-content-center mt-4"><%=@user.bio%></div>
+    </ul>
 
-
-<h1>All posts by  <%= @user.name %></h1>
-<p>
   <% @posts = Post.where(user_id: @user.id) %>
 </p>
 <ul>
     <% @posts.each do |post| %>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb]: - [ ] According to the requirements, you are supposed to use the `recent-posts` method in your user's model to show the 3 most recent posts. Here, you are showing all posts made by a user. Please fix that 👌 .

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @eaesmat  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Please make your `README.md` descriptive so that it talks about your project. It should look similar to [this](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍 

[OPTIONAL]
- I don't know if the problem is on my end but for some reason, nothing is rendering when I run this application. If its running well on your device please ignore this 👍 
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
-<p>
   <% @posts = Post.where(user_id: @user.id) %>
 </p>
 <ul>
-    <% @posts.each do |post| %>
-      <li>
-        <h4> <%= post.title %> </h4>
+    <% @posts.limit(3).each do |post| %>
+        <div class="container border rounded-3 p-3 col-8 justify-content-center mt-4">
+           <h4> <%= post.title %> </h4>
```
Comment (youmari) [inline app/views/users/show.html.erb]: - [ ]  as asked by the previous reviewer and also in the requirements you should use the methods that you created the last time to display the recent posts so you should use this [Method](https://github.com/eaesmat/blog/blob/81fc526b276dca7e5ac82dbed81ec3d13bc25295/app/models/user.rb#L8)
and try to change the code inside that method cause that code won't return the last recent posts so you change to something like `posts.order(created_at: 'desc').limit(3)`
- Screenshot of the requirements 
Screen Shot 2022-11-03 at 15 55 20

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @eaesmat , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @eaesmat      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x] Your project is professional 

- [x]  Your repo is professional 




 

- [x] No Linter Errors





Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**